test/kubemark/start-kubemark.sh

```diff
@@ -157,7 +157,7 @@ gcloud compute ssh --zone="${ZONE}" --project="${PROJECT}" "${MASTER_NAME}" \
     sudo bash -c \"echo ${password},admin,admin > /etc/srv/kubernetes/basic_auth.csv\""
 
 
-gcloud compute copy-files --zone="${ZONE}" --project="${PROJECT}" \
```

Comment:
I checked if both of them are working in the following way:
ssh: Added `sudo bash -c "if [[ $(($RANDOM % 2)) == 0 ]]; then exit 1;` to the list of commands in ssh, which makes the ssh fail randomly, but the code still works due to having 3 retries (with a probability of 1/8 and I was unlucky enough to get only this 1 case out of the 8 when I checked for the first time :p )
copy-files: Ran the same copy-files command back to back (but with the file content changed in between) and it worked as expected.